PsychSourceGL/Cohorts/FTGLTextRenderer/libptbdrawtext_ftgl.cpp

```diff
@@ -404,6 +404,10 @@ int PsychRebuildFont(fontCacheItem* fi)
         // Do not reset GL_MODELVIEW matrix after text drawing, so
         // we can read out the post-draw text cursor position from it:
         fi->faceT->setAdvance(true);
+
+	// set underlining, if requested in style
```

Comment:
Please use a comment style consistent with the existing one, e.g., a : (or . if you want) at the end of a sentence.
Also capital letter "Set underlining...", instead of "set underlining...", like the other english sentences.